streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -172,6 +174,11 @@ public void addStateStore(String stateStoreName) {
         public ProcessorNode build() {
             return new ProcessorNode<>(name, supplier.get(), stateStoreNames);
         }
+
+        @Override
+        TopologyDescription.Processor describe() {
```

Comment:
Any reason why we don't just stick with return `AbstractNode` from these overrides? I don't see anywhere that we need it to be a `Processor` etc